jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/local/SamRunningState.kt

```diff
@@ -53,6 +53,15 @@ class SamRunningState(
             }
         }
 
+        samOptions.additionalLocalArgs?.let {
+            if (it.isNotBlank()) {
+                val argParts = it.split(" ")
+                for (arg in argParts) {
```

Comment:
Can we get the same thing here:
`commandLine.withParameters(*it.split(" ").toTypedArray())`

Reply:
Oops forgot that one...